internal/filter/on_collab_style_test.go

```diff
@@ -0,0 +1,41 @@
+package filter_test
+
+import (
+	"github.com/clean-code-projects/co-coders-api/internal/assert"
+	"github.com/clean-code-projects/co-coders-api/internal/criteria"
+	"github.com/clean-code-projects/co-coders-api/internal/filter"
+	"github.com/clean-code-projects/co-coders-api/internal/user"
+
+	"testing"
+)
+
+func TestFilterOnCoStyleReturnsEmpty(t *testing.T) {
```

Comment:
We changed coStyle to colabStyle, but didn't update the test names.

Reply:
Yeah, I forgot to update them. I'll do that.